docs/core/compatibility/serialization/5.0/non-public-parameterless-constructors-not-used-for-deserialization.md

```diff
@@ -31,7 +31,7 @@ If none of these constructors are available, a <xref:System.NotSupportedExceptio
 ## Recommended action
 
 - If you own the type and it's feasible, make the parameterless constructor public.
-- Otherwise, implement a `JsonConverter<T>` for the type and control the deserialization behavior.
+- Otherwise, implement a <xref:System.Text.Json.Serialization.JsonConverter`1> for the type and control the deserialization behavior. You can call a non-public constructor from a <xref:System.Text.Json.Serialization.JsonConverter`1> implementation if C# accessibility rules for that scenario allow it.
```

Comment:
```suggestion
- Otherwise, implement a <xref:System.Text.Json.Serialization.JsonConverter%601> for the type and control the deserialization behavior. You can call a non-public constructor from a <xref:System.Text.Json.Serialization.JsonConverter%601> implementation if C# accessibility rules for that scenario allow it.
```